pds_doi_service/core/db/doi_database.py

```diff
@@ -344,27 +300,27 @@ def select_rows(self, query_criterias, table_name=None):
 
         query_string = f'SELECT * FROM {table_name}'
 
-        if len(query_criterias) > 0:
-            query_string += ' WHERE '
+        criterias_str, criteria_dict = DOIDataBase.parse_criteria(query_criterias)
```

Comment:
My "SQL Injection" Spidey-Sense is no longer going off 😎

Although now my English degree is going off…"criteria" is already plural 😆 (ignore me!)

Reply:
Thanks @nutjob4life , I always had a doubt on criteria being singular or plural since I had no idea what the singular was. I 've just checked, now I know, this is 'criterion', but 'criteria' is also used as a singular.... this is confusing... https://www.merriam-webster.com/words-at-play/criteria-vs-criterion-singular-plural-grammar#:~:text=Criteria%20is%20typically%20a%20plural,used%20singulars%20datum%20and%20agendum.

Reply:
Oh yeah, I guess it's like "data" being both as well. Thanks!